libnavigation-base/src/main/java/com/mapbox/navigation/base/model/route/RouteState.kt

```diff
@@ -0,0 +1,10 @@
+package com.mapbox.navigation.base.model.route
+
+enum class RouteState private constructor() {
```

Comment:
Why did we add this enum? Because in legacy `RouteProgressStateMap` we use `RouteState` provided by side library with package `com.mapbox.navigator`